apps/expo/src/app/privacy-policy/index.tsx

```diff
@@ -0,0 +1,225 @@
+              {'\n\t\t\t'} Barbell takes reasonable measures to protect your information from
+              unauthorized access, use, or disclosure. However, no method of transmission over the
+              internet or electronic storage is 100% secure. Therefore, we cannot guarantee absolute
+              security.
+              {'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              5. Your Choices and Rights
+            </Text>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n'}a. Account Settings:{'\n\t\t\t'} You have the option to review and update your
+              account information and privacy settings within the App.{'\n\n'}b. Opt-Out:
+              {'\n\t\t\t'} You may choose not to share certain information or opt-out of specific
+              features or services within the App.{'\n\n'}c. Access and Rectification:{'\n\t\t\t'}{' '}
+              You have the right to access and correct inaccuracies in your personal information
+              held by Barbell.
+              {'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              6. Children's Privacy
+            </Text>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n\t\t\t'}
+              Barbell is not intended for individuals under the age of 13. We do not knowingly
+              collect personal information from children under 13. If you are a parent or guardian
+              and believe that your child has provided us with personal information, please contact
+              us immediately.{'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              7. Changes to this Privacy Policy
+            </Text>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n\t\t\t'} We reserve the right to update or modify this Privacy Policy at any time.
+              We will notify you of any material changes by posting the updated policy on our
+              website or through other communication channels.{'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              8. Contact Us
```

Comment:
remove this section, we have a dedicated contact us page #229

Reply:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/943cb33c-1fbc-463e-83e9-5dc59dd10d76)

this is after I changed some styling 

compare to 

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/ae633bc1-e852-4c4c-9246-00cf33fb3eab)

Reply:
Done